file_system.go

```diff
@@ -112,6 +112,27 @@ func (c *Client) Getwd() (string, error) {
 	return dir, nil
 }
 
+// Setwd changes the current working directory.
+func (c *Client) Setwd(path string) (error) {
```

Comment:
I know but it seems it is used as a workaround by ftp clients when LIST or MLSD fails with path argument. I ran into the issue while testing and using CWD first before LIST (without path). 

I was considering the following:
* renaming all functions that call getIdleConn() to *Ex() and replace them with a wrapper. 
* the Ex() versions would get the idx of the pconn as a param 
* the Ex() versions would call getIdleConn() only if idx was -1 otherwise use the idx pconn / fail.
* SetWd() returns the pconn idx